templates/engage/thank-you.html

```diff
@@ -0,0 +1,58 @@
+{% extends "engage/base_engage.html" %}
+
+{% block title %}Engage download "{{ resource_name }}"{% endblock %}
+
+{% block head_extra%}<meta name="robots" content="noindex" />{% endblock %}
+
+{% block content %}
+<style>
+  .global-nav .global-nav__header-logo-anchor {
+    padding-left: 0 !important;
+  }
+</style>
+<section class="p-strip p-engage-banner--grad">
+  <div class="u-fixed-width navigation-logo-engage">
+    <a href="/">
+      {{
+        image(
+            url="https://assets.ubuntu.com/v1/f263d9c4-logo-ubuntu-white.svg",
+            alt="Ubuntu",
+            width="143",
+            height="32",
+            hi_def=True,
+            loading="auto",
+        ) | safe
+      }}
+    </a>
+  </div>
+  <div class="row">
+    <div class="col-8">
+      <div>
+        <h1>
+          Thank you
+        </h1>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row">
+    <div class="col-8">
+      {% if 'pages.ubuntu.com' in resource_url %}
+      <p>
+        <strong>{{ resource_name }}</strong> is now ready to download.
```

Comment:
Wouldn't this be better as something like:

```suggestion
        "{{ resource_name }}" is now ready to download.
```

Or:

```
<a class="p-button--positive" href="{{ resource_url }}">{{ resource_name }}</a> is now ready to download.
```